lib/puppet/file_serving/configuration.rb

```diff
@@ -81,6 +82,8 @@ def mk_default_mounts
     @mounts["modules"].allow('*') if @mounts["modules"].empty?
     @mounts["plugins"] ||= Mount::Plugins.new("plugins")
     @mounts["plugins"].allow('*') if @mounts["plugins"].empty?
+    @mounts["locales"] ||= Mount::Locales.new("locales")
```

Comment:
We do this with most everything else so its probably fine to keep doing this but one thing that I find bizarre is that this code, in theory, would let a user override the "locales" mountpoint with a custom setting in the fileserver.conf (as well as setting its auth to deny). Which seems like something we wouldn't want to allow (here or with any of the other special mounts).

Reply:
We also base the downloading itself on settings, so in theory you could point the locales downloader at a different mountpoint, which also seems weird to me.